binary_search/C/binary_search.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+
+int main()
+{
+	int arr[5] = { 1, 4, 5, 25, 50};
+	int left =0, right=4, mid =0,flag=0;
+	int search = 25;
+	while(left<right)
+	{
+		mid = (left+right)/2;
+		if(arr[mid]==search)
+		{	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmplw4hjhov/binary_search/C/binary_search.c
+++ b/tmp/tmplw4hjhov/binary_search/C/binary_search.c
@@ -9,7 +9,7 @@
 	{
 		mid = (left+right)/2;
 		if(arr[mid]==search)
-		{	
+		{
 			flag=1;
 			break;
 		}
```